### PR TITLE
Solving issue #2397 - issues with bot accounts being counted for room size

### PR DESCRIPTION
**Pull request by dqnk**
- add support for tracking bots in elixir database
- added counting of people and bots in rooms
- add fields for bot variables in the database
- add environment variables file in /kousa to .gitignore

I tested the PR and it shouldn't break anything, but I was unable to create bot accounts on my localhost to test if they actually don't count to final room count anymore #2397.

### CODE REVIEW

File: kousa/lib/beef/schemas/user.ex [hunk 1]
```diff
@@ -77,6 +78,7 @@ defmodule Beef.Schemas.User do
     field(:avatarUrl, :string)
     field(:bannerUrl, :string)
     field(:bio, :string, default: "")
+    field(:isBot, :boolean, default: false)
```
Comment (amitojsingh366) [inline kousa/lib/beef/schemas/user.ex]: we don't want this, we already have `botOwnerId` and we can track if a user is a bot or not by using that

**Review by amitojsingh366 — CHANGES_REQUESTED** [review]

File: kousa/lib/beef/schemas/user.ex (hunk 1 above)
Comment (dqnk) [reply kousa/lib/beef/schemas/user.ex]: thanks for noticing, will fix

Comment (dqnk) [reply kousa/lib/beef/schemas/user.ex]: So if I understand correctly, botOwnerId (in `kousa/lib/beef/schemas/user.ex`) is either `null/nil` for regular accounts or an `Ecto` type for bots?

Comment (amitojsingh366) [reply kousa/lib/beef/schemas/user.ex]: also we probably don't wanna do this at the backend, this can maybe be done fully on the client side

**Comment by amitojsingh366** [comment]
we probably don't wanna do this at the backend and maybe do it client side only but that's upto @benawad

**Comment by amitojsingh366** [comment]
also please use proper commit messages *(`yarn commit` or `yarn commit:signed`)*

**Comment by amitojsingh366** [comment]
and your elixir tests are failing

**Comment by benawad** [comment]
idk if I want this behavior, I will think on it

**Comment by dqnk** [comment]
Note that my last commit probably doesn't solve all the issues with elixir.

**Comment by amitojsingh366** [comment]
> idk if I want this behavior, I will think on it

so do we close this then

**Comment by benawad** [comment]
I'm going to close this for now because we are going to be moving the # of users count out of the database

**Pull request closed without merging**